### ISSUE TITLE
News (Listing feed from news section on forum, based on RSS)

### ISSUE THREAD

**Issue by sigdestad** [enonic/site-developer#641]

**Comment by reisfmb** [comment]
@poi33 @sigdestad @alansemenov 

Here are two initial suggestions:

![devportal-news-section-1st-suggestion](https://user-images.githubusercontent.com/67838246/152858484-7724f7ea-fa0a-4b52-98c4-52e60cf292b6.png)

![devportal-news-section-2nd-suggesiton](https://user-images.githubusercontent.com/67838246/152858112-d64463cc-76bd-46a9-9e1d-82c9e96c6d8b.png)

**Comment by poi33** [comment]
I like both suggestions. 
The second one maybe a little more. 
Did you take a look at discuss or rss feed on what information is available? Would be nice to see what info is available.

**Comment by alansemenov** [comment]
I like the second one but would like even smaller font for the heading, and some kind of highlight (bigger font or different color) on app name.

**Comment by alansemenov** [comment]
Ah, that's heading that comes directly from Discuss as-is... Ok, then I guess we cannot highlight only parts of it.

**Comment by sigdestad** [comment]
I like the first, it is a bit confusing with the blue like varying in size? We can ofc cut down the lenght of the text even more..
BTW: How do you plan to implement this, so page does not hang in case Discuss is down?

**Comment by reisfmb** [comment]
> I like both suggestions. The second one maybe a little more. Did you take a look at discuss or rss feed on what information is available? Would be nice to see what info is available.

Yes, I did the design based on what was available... Here it is: [https://discuss.enonic.com/c/news/5.rss](https://discuss.enonic.com/c/news/5.rss)

**Comment by reisfmb** [comment]
@poi33 @sigdestad @alansemenov 

First version of the **News Part**!

**Desktop:**
![image](https://user-images.githubusercontent.com/67838246/153655695-1e6ce339-a274-4570-a8c3-29a2ef2b881d.png)

**Mobile:**
![image](https://user-images.githubusercontent.com/67838246/153655842-966898ea-b80c-471a-abd2-9a03190603b8.png)

**Part config.:**
![image](https://user-images.githubusercontent.com/67838246/153655919-dac49be0-d4f5-4166-a4a2-629dc22050ea.png)

**Potential issue(s):**
Currently uses `lib-cache` with 1 hour as expiration time... that implies that the controller will eventually handle new requests to the rss feed and the page load might be affected in those cases. 

For the second version, as @alansemenov suggested, I also think that I'd be important to implement a cache layer that is populated by some task which is executed periodically by a scheduler.

**Comment by reisfmb** [comment]
First version is in [QA](https://qa.enonic.com/admin/tool/com.enonic.app.contentstudio/main#/dev-portal/edit/4b94f33c-2585-4355-90d2-32363408f523).
